docs/ru/sql-reference/data-types/datetime64.md

```diff
@@ -9,12 +9,12 @@ sidebar_label: DateTime64
 Позволяет хранить момент времени, который может быть представлен как календарная дата и время, с заданной суб-секундной точностью.
 
 Размер тика (точность, precision): 10<sup>-precision</sup> секунд, где precision - целочисленный параметр. Возможные значения: [ 0 : 9 ].
-Обычно используются - 3 (миллисекунды), 6 (микросекунды), 9 (наносекунды).
```

Comment:
Could you please edit the english version in the same way?

Reply:
Im sorry. I know english not wery well to write documentation.

The DateTime64 has default precision 3. It is all that i wrote.